assets/xml/objects/gameplay_keep.xml

```diff
@@ -1,77 +1,90 @@
+        <DList Name="gKokiriSwordBladePart1DL" Offset="0x2838" />
         <DList Name="gKokiriSwordBladeDL" Offset="0x28C0" />
-        <Texture Name="gameplay_keep_TLUT_002908" OutName="tlut_002908" Format="rgba16" Width="16" Height="16" Offset="0x2908" />
-        <Texture Name="gameplay_keep_Tex_002B08" OutName="tex_002B08" Format="rgba16" Width="8" Height="8" Offset="0x2B08" />
-        <Texture Name="gameplay_keep_Tex_002B88" OutName="tex_002B88" Format="rgba16" Width="16" Height="16" Offset="0x2B88" />
-        <Texture Name="gameplay_keep_Tex_002D88" OutName="tex_002D88" Format="ci8" Width="8" Height="8" Offset="0x2D88" />
-        <Texture Name="gameplay_keep_Tex_002DC8" OutName="tex_002DC8" Format="i8" Width="8" Height="8" Offset="0x2DC8" />
-        <Texture Name="gameplay_keep_Tex_002E08" OutName="tex_002E08" Format="rgba16" Width="16" Height="16" Offset="0x2E08" />
-        <Texture Name="gameplay_keep_Tex_003008" OutName="tex_003008" Format="i8" Width="8" Height="8" Offset="0x3008" />
+        <Texture Name="gKokiriSwordHandleTLUT" OutName="kokiri_sword_handle_tlut" Format="rgba16" Width="16" Height="16" Offset="0x2908" />
+        <Texture Name="gKokiriSwordHandlePommelTex" OutName="kokiri_sword_handle_pommel" Format="rgba16" Width="8" Height="8" Offset="0x2B08" />
+        <Texture Name="gRazorSwordHandleGripTex" OutName="razor_sword_handle_grip" Format="rgba16" Width="16" Height="16" Offset="0x2B88" />
+        <Texture Name="gKokiriSwordHandleCrossGuardTex" OutName="kokiri_sword_handle_cross_gaurd" Format="ci8" Width="8" Height="8" Offset="0x2D88" />
+        <Texture Name="gKokiriSwordHandleGripTex" OutName="kokiri_sword_handle_grip" Format="i8" Width="8" Height="8" Offset="0x2DC8" />
+        <Texture Name="gRazorSwordHandleDesignTex" OutName="razor_sword_handle_design" Format="rgba16" Width="16" Height="16" Offset="0x2E08" />
+        <Texture Name="gSwordMetalTex" OutName="sword_metal" Format="i8" Width="8" Height="8" Offset="0x3008" />
         <DList Name="gKokiriSwordDL" Offset="0x3050" />
         <DList Name="gRazorSwordDL" Offset="0x3060" />
+
         <DList Name="gDekuStickDL" Offset="0x32B0" /> <!-- @bug Supplies the wrong size to gsDPLoadTextureBlock, meaning it uses part of gLinkPauseChildJointTable as if it were a texture. -->
-        <Blob Name="gameplay_keep_Blob_003400" Size="0x9D0" Offset="0x3400" />
-        <DList Name="gameplay_keep_DL_005360" Offset="0x5360" />
 
+        <Array Name="gUnknownMaskVtx" Count="157" Offset="0x3400"> <!-- Based on the shape and TLUT this looks to be a remnant of the Adult Link mask -->
+            <Vtx/>
+        </Array>
+
+        <!-- Transformation Masks -->
+        <DList Name="gZoraMaskDL" Offset="0x5360" />
         <DList Name="gDekuMaskDL" Offset="0x56C0" />
+        <DList Name="gGoronMaskDL" Offset="0x5A10" />
+        <Texture Name="gUnkownMaskTLUT" OutName="unknown_mask_tlut" Format="rgba16" Width="16" Height="16" Offset="0x5DB8" />
```

Comment:
```suggestion
        <Texture Name="gUnknownMaskTLUT" OutName="unknown_mask_tlut" Format="rgba16" Width="16" Height="16" Offset="0x5DB8" />
```

Reply:
Based on the renders in discord, it seemed appropriate to rename them to `gLinkMaskVtx` and `gLinkMaskTLUT`. I've also added a comment saying rendering the vertices looks like the Link Mask in `object_stk`